docs/reference/datatiers.asciidoc

```diff
@@ -85,12 +85,14 @@ For resiliency, indices in the cold tier can rely on
 [[frozen-tier]]
 === Frozen tier
 
+experimental::[]
+
 Once data is no longer being queried, or being queried rarely, it may move from the cold tier
 to the frozen tier where it stays for the rest of its life.
-The frozen tier is a less responsive query tier than the cold tier, and data in the frozen tier is
-not normally updated. As data transitions into the frozen tier it can be compressed and shrunken.
-For resiliency, indices in the frozen tier can rely on <<ilm-searchable-snapshot, searchable
-snapshots>>, eliminating the need for replicas or even a local copy.
+The frozen tier is a less responsive query tier than the cold tier, and data in the frozen tier
+cannot be updated. The frozen tier holds searchable snapshots mounted using the
+`shared_cache` storage option exclusively. The <<ilm-index-lifecycle, frozen phase>> automatically converts data
```

Comment:
```suggestion
`shared_cache` storage option exclusively. The <<ilm-index-lifecycle, frozen phase>> converts data
```

(since technically we still have to specify the `searchable_snapshot` action in the phase, even if it is required)